### PR TITLE
Updating README with my name as a contributor

### LINKED ISSUE #8: My great user story

### PR DESCRIPTION
**Pull request by typeoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Contributors
+Shauna Griffin
+
```
Comment (Yanal-Sabbagh07) [inline README.md:3]: nice name what does it mean ?

**Comment by Wajih-alzouhairy** [comment]
it was nice work

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Contributors
+Shauna Griffin
```
Comment (momo0999) [inline README.md:2]: This is great

**Review by momo0999 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Yanal-Sabbagh07) [inline README.md:2]: ..

**Pull request merged**